test/mocks/SemanticjQuery-mock.js

```diff
@@ -19,6 +21,7 @@ let jQueryPlugins = {
   popup: sandbox.stub().returnsThis(),
   checkbox: sandbox.stub().returnsThis(),
   modal: sandbox.stub().returnsThis(),
+  transition: sandbox.stub().returnsThis(),
```

Comment:
Message uses this `transition()` jQuery plugin.  It is now added to our jQuery mock so we can test that it was called.  See `Message-test.js`.